scripts/build/version_info.js

```diff
@@ -0,0 +1,80 @@
+// GATHER VERSION INFO FROM COMPONENTS
+const path = require('path');
+const fs = require('fs');
+const {cwd} = process;
+
+// helpers
+const clean = name => name.replace(/^@axa\-ch\//, 'axa-');
+
+const justVersionInfo = info => info.replace(/[^A-Za-z0-9\.]/g, ''); // deliberate consequence: ignore caret
+
+const pick = (object, ignore = {}) => {
+  const result = {};
+  const components = Object.keys(object).filter( key => key.indexOf('@axa-ch/') === 0 && !ignore[key]);
+  components.forEach( name => {
+    result[clean(name)] = justVersionInfo(object[name]);
+  });
+  return result;
+}
+
+const ignoreNonComponents = (packages, ignores) => packages.filter( package => {
+    // ignore components classified under our Atomic-Design scheme as starting with 0
+    // (i.e. currently 00-materials, 05-utils)
+    const ignore = package.indexOf('src/components/0') > -1;
+    if (ignore) {
+        const [_, lastPathComponent] = package.match(/([a-z]+)$/);
```

Comment:
@markus-walther i guess `obj[str.length]` would be more readable but is fine as well like this

Reply:
I find mine a little bit more idiomatic because the first match is always the entire string covered by regexp, and `_` is the customary variable to say: I ignore this value.

So, since you leave it open here, I'd like to stick with the present code.